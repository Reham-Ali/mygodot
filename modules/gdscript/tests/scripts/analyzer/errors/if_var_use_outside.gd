func test():
    if var x = 100:
        print("t")

    print(x)
