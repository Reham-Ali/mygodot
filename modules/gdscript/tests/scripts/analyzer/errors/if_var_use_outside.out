GDTEST_ANALYZER_ERROR
Identifier "x" not declared in the current scope.
